assets/stylesheets/components/_header-box.scss

```diff
@@ -0,0 +1,26 @@
+.header-box {
```

Comment:
When naming this, follow the conventions we're using elsewhere such as `.section-header`, `.component-header` etc.